src/Tile.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react'
+import { Box, Span, theme } from '@truework/ui'
+import styled from 'styled-components'
+import { Field, FieldConfig, FieldProps } from 'formik'
+
+export type TileProps = {
+  id: string
+  icon?: React.ReactElement
+  label: string
+} & React.InputHTMLAttributes<HTMLInputElement>
+
+export type TileFieldProps = { name: string } & TileProps &
+  Pick<FieldConfig, 'validate'>
+
+const TileButton = styled.label(
+  ({ theme }) => `
+    width: 100%;
+    height: 100%;
+    position: relative;
+    display: flex;
+    justify-content: center;
+    cursor: pointer;
+`
+)
+
+const TileContent = styled.div`
+  width: 100%;
+  height: 100%;
+  padding: ${theme.space.med} ${theme.space.sm};
+  top: 0px;
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  color: ${theme.colors.body};
+  background-color: ${theme.colors.background};
+  border: 1px solid ${theme.colors.outline};
+  border-radius: ${theme.space.xxs};
+
+  transition-property: color, background-color, border-color;
+  transition-duration: ${theme.transitionDurations.fast};
+  transition-timing-function: ${theme.transitionTimingFunctions.ease};
+`
+
+const Input = styled.input`
+  margin: 0;
+  padding: 0;
+  appearance: none;
+
+  &:checked
+    ~ ${TileContent},
+    &:hover
+    ~ ${TileContent},
+    &:focus
+    ~ ${TileContent} {
+    border: 1px solid ${theme.colors.primary};
+    background-color: rgb(91, 99, 254, 0.1);
+  }
+
+  &:checked ~ ${TileContent} ${Span} {
+    color: ${theme.colors.primary};
+  }
+`
+
+export function Tile ({ id, icon, label, width, height, ...props }: TileProps) {
+  return (
+    <TileButton htmlFor={id}>
+      <Input id={id} type='radio' {...props} />
+
+      <TileContent>
+        <Box display='flex' alignItems='center' justifyContent='center' mb='xs'>
```

Comment:
Since you have `flex-direction: column` above, I don't think you don't need these flex props here anymore.

```suggestion
        <Box mb='xs'>
```